### ISSUE TITLE
Zip file blocked by Chrome

### ISSUE THREAD

**Issue by monicaca** [EragonJ/Kaku#190]
![11209435_10153453079755140_4628593479399293184_n](https://cloud.githubusercontent.com/assets/13413369/8771218/0477f38e-2ef1-11e5-9639-b9239e378588.jpg)

After click on the github link from Kaku Facebook fanpage and download the zip file, Chrome seemed to block it.

**Comment by EragonJ** [comment]
Interesting ... I'll try to figure out what's going on to make Chrome block it, thanks for the report !

**Comment by mimi89999** [comment]
@EragonJ Chrome blocks .zip with .exe in it.

**Comment by EragonJ** [comment]
@mimi89999 sad

**Comment by EragonJ** [comment]
This issue should be postponed until we have windows installer.

**Comment by EragonJ** [comment]
Close first due to low priority.
